internal/store/statefulset.go

```diff
@@ -178,6 +178,26 @@ func statefulSetMetricFamilies(allowAnnotationsList, allowLabelsList []string) [
 				}
 			}),
 		),
+		*generator.NewFamilyGeneratorWithStability(
+			"kube_statefulset_ordinals_start",
+			"Start ordinal of the StatefulSet.",
+			metric.Gauge,
+			basemetrics.ALPHA,
+			"",
+			wrapStatefulSetFunc(func(s *v1.StatefulSet) *metric.Family {
+				ms := []*metric.Metric{}
+
+				if s.Spec.Ordinals != nil {
```

Comment:
What if .Ordinals exists but .Ordinals.Start does not?

Reply:
.Ordinals is a pointer, while .Ordinals.Start is an integer. So .Ordinals.Start will default to zero. The user doesn't have a way to set .Ordinals, without specifying Start.